bin/make_view

```diff
@@ -96,4 +96,24 @@ CREATE VIEW combined_contributions AS
   LEFT OUTER JOIN oakland_candidates
   ON "FPPC"::varchar = "Filer_ID"
   WHERE "Start_Date" IS NULL OR "Tran_Date" >= "Start_Date";
+
+DROP VIEW IF EXISTS combined_independent_expenditures;
+CREATE VIEW combined_independent_expenditures AS
+  SELECT "FPPC" AS "Filer_ID", "Filer_NamL", "Exp_Date", "Sup_Opp_Cd", "Amount"
+  FROM (
+    SELECT "Filer_NamL", "Exp_Date", "Cand_NamF", "Cand_NamL", "Amount", "Sup_Opp_Cd"
+    FROM "496"
+    WHERE "Cand_NamL" IS NOT NULL
+    UNION ALL
+    SELECT "Filer_NamL", "Expn_Date" as "Exp_Date", "Cand_NamF", "Cand_NamL",
+    "Amount", "Sup_Opp_Cd"
+    FROM "D-Expenditure"
+    WHERE "Cand_NamL" IS NOT NULL
+      AND "Expn_Code" = 'IND'
+  ) AS all_data
+  JOIN "oakland_candidates"
+  ON LOWER(TRIM(CONCAT("Cand_NamF", ' ', "Cand_NamL"))) = LOWER("Candidate")
+    OR LOWER("Aliases") LIKE LOWER(CONCAT('%', TRIM(CONCAT("Cand_NamF", ' ', "Cand_NamL")), '%'))
+  WHERE ("Start_Date" IS NULL OR "Exp_Date" >= "Start_Date")
+  AND "FPPC" IS NOT NULL
```

Comment:
Can we push this to the inner queries, or alternatively move the "Cand_NamL" clause to the outer query, just to have all the filtering logic in one place?

Reply:
This?  FPPC? That's a column in the joined table. I guess we could move Cand_NamL.  I was thinking that you want to filter early, but maybe I shouldn't over guess the query optimizer.